src/views/virtualmachinesinstance/details/tabs/details/utils/VirtualMachinesInstancePageDetailsTabUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { V1CPU } from '@kubevirt-ui/kubevirt-api/kubevirt';
```

Comment:
@metalice I we can use these selectors/utils in other places like the wizard's scheduling tab.
please move those to the `src/utils`

Reply:
moved to utils